functional/FunctionalTest.java

```diff
@@ -445,97 +436,98 @@ public static void putObject_test2() throws Exception {
+   * Test: putObject(String bucketName, String objectName, InputStream body, long
+   * size, String contentType).
    */
   public static void putObject_test4() throws Exception {
     if (!mintEnv) {
       System.out.println("Test: putObject(String bucketName, String objectName, InputStream body, "
-                        + "long size, String contentType)");
+          + "long size, String contentType)");
     }
 
     long startTime = System.currentTimeMillis();
     try {
       String objectName = getRandomName();
-      try (final InputStream is = new ContentInputStream(1 * MB)) {
-        client.putObject(bucketName, objectName, is, 1 * MB, customContentType);
+      try (final InputStream is = new ContentInputStream(1 * KB)) {
+        client.putObject(bucketName, objectName, is, 1 * KB, customContentType);
       }
 
       client.removeObject(bucketName, objectName);
-      mintSuccessLog("putObject(String bucketName, String objectName, InputStream body, long size,"
-                     + " String contentType)",
-                     "size: 1 MB, objectName: " + customContentType, startTime);
+      mintSuccessLog(
+          "putObject(String bucketName, String objectName, InputStream body, long size," + " String contentType)",
+          "size: 1 KB, objectName: " + customContentType, startTime);
     } catch (Exception e) {
       mintFailedLog("putObject(String bucketName, String objectName, InputStream body, long size, String contentType)",
-                    "size: 1 MB, objectName: " + customContentType, startTime, null,
-                    e.toString() + " >>> " + Arrays.toString(e.getStackTrace()));
+          "size: 1 KB, objectName: " + customContentType, startTime, null,
+          e.toString() + " >>> " + Arrays.toString(e.getStackTrace()));
       throw e;
     }
   }
 
   /**
-   * Test: putObject(String bucketName, String objectName, InputStream body, long size, String contentType).
-   * where objectName has multiple path segments.
+   * Test: putObject(String bucketName, String objectName, InputStream body, long
+   * size, String contentType). where objectName has multiple path segments.
    */
   public static void putObject_test6() throws Exception {
     if (!mintEnv) {
       System.out.println("Test: objectName with path segments: "
-                        + "putObject(String bucketName, String objectName, InputStream body, "
-                        + "long size, String contentType)");
+          + "putObject(String bucketName, String objectName, InputStream body, " + "long size, String contentType)");
     }
 
     long startTime = System.currentTimeMillis();
     try {
       String objectName = "/path/to/" + getRandomName();
-      try (final InputStream is = new ContentInputStream(3 * MB)) {
-        client.putObject(bucketName, objectName, is, 1 * MB, customContentType);
+      try (final InputStream is = new ContentInputStream(1 * KB)) {
```

Comment:
The stream size should be greater than `1 * KB`

Reply:
why?